client/src/components/App.module.css

```diff
@@ -1,6 +1,56 @@
 .container{
-  width: 1000px;
+  width: 1100px;
   position: center;
-  padding-left: 200 px;
-  padding-right: 200 px;
+  padding-left: 40 px;
+  padding-right: 40 px;
+}
+
+
+
+.showAllBtn{
```

Comment:
Using !important in this context may not be necessary. It is not forbidden in the AirBNB style guide, so I will leave it at your discretion, but I have generally heard recommendations to avoid its usage if possible.

Reply:
Awesome thanks for the recommendation.